Cargo.toml

```diff
@@ -16,7 +16,7 @@ digest = "0.10.6"
 hex = "0.4.3"
 miette = "5.7.0"
 serde = { version = "1.0.152", optional = true }
-sha-1 = "0.10.1"
+sha-1 = "^0.10"
```

Comment:
isn't `"0.10.0"` the same thing (also without the `^`? I prefer having a semver-patch version in my deps.

Reply:
ah it might be, I haven't looked at the version spec in a while